.agents/skills/create-pr/SKILL.md

```
@@ -0,0 +1,127 @@
+2. Confirm every staged file belongs to the requested change.
+3. Use a Conventional Commit message:
+
+```text
+feat(scope): ...
+fix(scope): ...
+chore(scope): ...
+docs(scope): ...
+test(scope): ...
+```
+
+4. Commit only after the staged diff and verification are clean.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)